File: bookhub-back/src/main/java/com/example/bookhub_back/dto/book/response/BookLogResponseDto.java
package com.example.bookhub_back.dto.book.response;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BookLogResponseDto {
    private Long bookLogId;
    private String bookIsbn;
    private String bookTitle;
    private String bookLogType;
    private Long previousPrice;
    private Long previousDiscountRate;
    private String employeeName;
    private Long policyId;
    private LocalDateTime changedAt;
}

File: bookhub-front/src/dtos/book/response/BookLog.response.dto.ts
import { BookLogType } from "@/apis/enums/BookLogType";

export interface BookLogResponseDto {
  bookLogId: number;
  bookIsbn: string;
  bookTitle: string;
  bookLogType: BookLogType;
  previousPrice?: number;
  previousDiscountRate?: number;
  employeeName: string;
  policyId?: number;
  changedAt: Date;
}